corehq/apps/reports_core/static/reports_core/js/maps.js

```diff
@@ -37,30 +52,30 @@ var maps = (function() {
             });
             var table = '<table class="table table-bordered"><%= rows %></table>';
             var template = _.template(table)({rows: rows.join('')});
-            fn.template = _.template(template);
+            privates.template = _.template(template);
```

Comment:
btw, that was just the quickest example I could find of how this `private` member is bleeding state between contexts. There may be others.